seshdash/views.py

```diff
@@ -669,20 +669,27 @@ def silence_alert(request):
 
 @login_required
 def get_latest_bom_data(request):
```

Comment:
can you make this configurable?
So, it could accept a array  if needed with the measurment names in it?
This way we can use this to generate many types of other cards if needed.
We can start of with a default list which is what the status card is today